### ISSUE TITLE
action_mw, action_mwl ignores 'sender'

### ISSUE THREAD

**Issue by iGeorgeX** [fail2ban/fail2ban#1075]
In **/etc/fail2ban/jail.conf** 

**action_mw** and **action_mwl**

ignores completely the **sender**, only **action_xarf** includes the **sender**, this way I can't change the sender for **action_mw** and **action_mwl** from which fail2ban sends eMails. So I changed for **action_mwl** (works the same way for **action_mw**) the line

```
[name=%(__name__)s, dest="%(destemail)s", logpath=%(logpath)s, chain="%(chain)s"]
```

to

```
[name=%(__name__)s, dest="%(destemail)s", sender="%(sender)s", logpath=%(logpath)s, chain="%(chain)s"]
```

and this fixed the problem for me.

GM

**Comment by mitchellkrogza** [comment]
Thanks for that, I ended up fiddling with the mail.conf file which did nothing for me and then ended up modifying my  	mail-whois-lines.conf to include the -a "From: Fail2Ban <security@allover.co.za>" in the mail command line which worked but I will test out your solution instead to see if it obeys me.

**Comment by sebres** [comment]
Was already fixed in 40c0bed82c649ad7731bbe33ba773a25e55a6163 (ver. 0.9.4)
